### DISCUSSION TITLE
Hobbies and interests

### DISCUSSION THREAD

**Discussion by aklidata** [PullDating/Org#22] [Ideas & Features]
### The case for making hobbies and interests an integral part of engagement:

Many Dating apps allow users to add their hobbies/interests into their profile. Some use it to help the algorithm match people correctly and some use it just as a profile show case. 

My idea is to create engagement around common interests; The rise of Facebook groups and growth of Reddit shows a trend that people engage well with group minded media, where they can communicate with other who share their interests. 

However in dating apps it leaves the interaction up to the user. Perhaps they drop a pick up line that references someone's tagged hobby, perhaps they overlook the possibility all together. 

We should think up a way to create some kind of group mentality around shared interests or maybe lobbies/better curated areas of shared interest.

**Comment by WilliamDormer** [comment]
I like this idea. I think that making groups around shared interests would make it easy to get people to find common ground for dates etc. I do want to be careful thought about assuming that people that have similar interests and hobbies could only match with the same type of people.

**Reply by aklidata** [reply]
Agreed on your last point, however its a good ice breaker if paired with other metrics it could prove valuable.

**Comment by ajbott** [comment]
I think being able to bond over shared interests would be a good kickstart in bonding! It could also kickstart some conversations in learning about their interests your interested in.
